qiita_db/metadata_template/prep_template.py

```diff
@@ -726,13 +726,15 @@ def modification_timestamp(self):
     def max_samples():
         return qdb.util.max_preparation_samples()
 
-    def add_default_workflow(self, user):
+    def add_default_workflow(self, user, workflow=None):
```

Comment:
Sorry to ask this, but why would we ever call a function add_default_workflow() and not specify a workflow? Having a default of None seems very strange.

Reply:
I see what you are saying, my guess is missing some context: basically this will add the commands in a default workflow (definition) to a preparation (thus, it lives in qiita_db/metadata_template/prep_template.py) and before these changes it would just create a new workflow (object) and add the commands of the default workflow (definition); and now if a workflow (object) is passed it will not create a new one but use the object passed. Does this make sense?

Reply:
Thank you for the explanation! I see what you're saying. I think adding this as a comment would be very helpful to future readers of the code. Perhaps changing the name to update_default_workflow() might be good too, maybe?

Reply:
I don't agree with that name cause it doesn't only update but adds/creates a new workflow; any other recommendation?

Reply:
maybe just having comment is best, yes?